src/internal-server/app/Controllers/Http/UserController.js

```diff
@@ -304,25 +299,17 @@ class UserController {
         .rows.length > 0
     );
 
-    if (
-      workspace.services &&
-      typeof workspace.services === 'string' &&
-      workspace.services.length > 0
```

Comment:
This is no longer necessary?

Reply:
thanks - must have slipped my sight. will fix in new commit

Reply:
that's already handled in the `convertJSON` method